lutris/util/desktopapps.py

```diff
@@ -0,0 +1,127 @@
+"""Get games installed as desktop applications."""
+
+import os
+import shutil
+import shlex
+import subprocess
+import re
+
+from gi.repository import Gio
+from lutris import pga
+from lutris.util.log import logger
+from lutris.util.strings import slugify
+from lutris.config import make_game_config_id, LutrisConfig
+
+IGNORED_APPS = (
+    "lutris", "mame", "dosbox", "playonlinux", "org.gnome.Games", "retroarch",
+    "steam", "steam-runtime", "steam-valve", "steam-native"
+)
```

Comment:
GNOME Games also use [blacklists](https://github.com/GNOME/gnome-games/tree/master/plugins/desktop/data/blacklists), I'm not sure how this could work otherwise.
Maybe we can have a menu item Tools > Import games > Installed and present a list of games and let the user choose what to not import.